tests/misc/test_file_server.py

```diff
@@ -17,3 +17,14 @@ def test_fileserver_redirect_https(host):
     lines = stdout.decode().splitlines()
     assert lines[0].strip() == "HTTP/1.1 301 Moved Permanently"
     assert lines[2].strip() == "location:https://" + host.hostname_or_ip + path
+
+def test_fileserver_hsts(host):
+    process = subprocess.Popen(
+        ["curl", "-XGET", "-i", "https://" + host.hostname_or_ip],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+    stdout, _ = process.communicate()
+    lines = stdout.decode().splitlines()
+    assert lines[0].strip() == "HTTP/1.1 200 OK"
+    assert lines[3].strip() == "Strict-Transport-Security:max-age=63072000; includeSubDomains; preload"
```

Comment:
I'm not sure we want `includeSubDomains` and even less `preload`.

Reply:
Agree with preload. I'm not sure either.  But if we want to enforce the usage of HTTPS I think that `includeSubDomains` makes sense.

Reply:
Actually we can't presume of how a XCP-ng user will use the subdomains so better be safe and only use the age.